src/app/api/v1/hibp/notify/route.ts

```diff
@@ -108,7 +108,7 @@ export async function POST(req: NextRequest) {
     pubsub = new PubSub({ projectId });
   } catch (ex) {
     logger.error("Error connecting to PubSub:", ex);
-    return NextResponse.json({ success: false }, { status: 500 });
+    return NextResponse.json({ success: false }, { status: 429 });
```

Comment:
Cool, first time I'm seeing 429 Too Many Requests in use! So I assume this is usually a rate-limiting error?

Reply:
Yes that's right, currently we alert based on HTTP 500s (from the loadbalancer logs) so 429 is a more useful signal both for us and for the caller.

Reply:
If we want to get fancier with it we can include a https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Retry-After header as well, and implement actual rate limiting.